conjur-api/ResourceMetadata.cs

```diff
@@ -59,4 +59,16 @@ public class Annoatation
         public string UpdatedAt { get; private set; }
     }
 
+    [DataContract]
+    public class RoleMember
+    {
+        [DataMember(Name = "admin_option")]
+        public bool Admin { get; set; }
```

Comment:
Can we keep this name aligned with other members (admin != admin_options)? e.g UpdatedAt.